std/format.d

```diff
@@ -4052,103 +4054,96 @@ here:
     const string[] widths = [ "", "0", "4", "20" ];
     const string[] precs = [ "", ".", ".0", ".4", ".20" ];
     const string formats = "sdoxXeEfFgGaA";
-  /+
```

Comment:
Why was this commented out in the first place? What is the rationale for putting it back in?